tracery/modifiers.py

```diff
@@ -67,3 +67,130 @@ def lowercase(text, *params):
     'uppercase': uppercase,
     'lowercase': lowercase
 }
+
+
+#----
+#inflect.py modifiers
+#Mappings based on https://github.com/pwdyson/inflect.py
+
+import inflect
+p = inflect.engine()
+
+
+def ordinal(text, *params):
+    return p.ordinal(text)
+
+def number_to_words(text, *params):
+    kwargs=dict(param.split('=') for param in params)
+    try:
+        resp = p.number_to_words(text, **kwargs)
+    except:
```

Comment:
I don't want `except` without an exception type here (since it might catch exceptions unrelated to the specific error you're looking for). why not simply remove the `try`/`except` and let the underlying function from `inflect` just throw the error?